internal/service/ipam/model_ipamsvc_range.go

```diff
@@ -84,6 +84,7 @@ var IpamsvcRangeResourceSchemaAttributes = map[string]schema.Attribute{
 	},
 	"dhcp_host": schema.StringAttribute{
 		Optional:            true,
+		Computed:            true,
```

Comment:
why was this marked computed?

Reply:
when creating FA under range the parent id is inherited.